src/components/navigation/NavigationMenu.vue

```diff
@@ -26,11 +26,11 @@
                             <span class="tagged-link__content margin-right margin-right--half-width" data-ref="serverlist" @click.prevent.stop="emitClick($event.target)">Server List</span>
                         </a>
                     </li>
-                    <li><a href="#" @click="launchModded"><i class="fas fa-play-circle icon--margin-right"/>Start modded</a>
+                    <li>
+                        <a href="#" @click="launch(true)"><i class="fas fa-play-circle icon--margin-right"/>Start modded</a>
```

Comment:
Not a huge fan of true/false for arguments. Best to be explicit so a better option could be an enum if you don’t like multiple method separation.

Reply:
Agreed 👍